### ISSUE TITLE
External login error on new users

### ISSUE THREAD

**Issue by elgransan** [oqtane/oqtane.framework#2503]
The default password generated is not complying the password policy and fails

![image](https://user-images.githubusercontent.com/9949584/202790621-40850dff-5806-40d5-a8be-b50d822abe10.png)

**Comment by sbwalker** [comment]
Did you change the default password policy in your installation?

**Comment by elgransan** [comment]
no, it was a fresh install where I was trying to test openid connect, I solved removing the second parameter.

**Comment by sbwalker** [comment]
In North America the statement DateTime.UtcNow.ToString("yyyy-MMM-dd-HH-mm-ss") will produce a result of "2022-Nov-23-10-57-00" - so 20 characters in length with upper case characters, lower case characters, digits, and punctuation - which will pass the default password complexity criteria. 

So I am guessing this is a Locale issue - most likely the MMM is not producing a month with upper and lower case characters? I will need to do some research.

**Comment by elgransan** [comment]
Ok, it's that:  2022-nov.-23-19-31-34 in Spanish

**Comment by sbwalker** [comment]
It should be fixed with the PR #2506

**Comment by elgransan** [comment]
cool thanks!
